R/workflow_functions.R

```diff
@@ -85,7 +85,8 @@ get_edi_file <- function(edi_https, file, lake_directory){
     }
     download.file(edi_https,
                   destfile = file.path(lake_directory, "data_raw", file),
-                  method="curl")
+                  method="curl",
+                 extra = options(timeout = 120))
```

Comment:
Can you make the timeout an option argument to the get_edi_file function?  Have it default to 60 seconds.  We can then update the function call in all the individual lake repos as they time out.  This current approach requires us to update FLAREr each time we want to change the timeout